docs/src/main/asciidoc/_admin-guide/subsystem-configuration/EJB3.adoc

```diff
@@ -266,3 +266,52 @@ public class FooInterceptor {
     }
 }
 ----
+
+[[client-interceptors]]
+== <client-interceptors>
+
+This element configures a number of client-side interceptors which can be
+configured without changing the deployments.
+
+Each interceptor is configured in _<interceptor>_ tag which contains the
+following fields:
+
+* `module` - the module in which the interceptor is defined
+* `class` - the class which implements the interceptor
+
+In order to use server interceptors you have to create a module that implements
+them and place it into _${WILDFLY_HOME}/modules_ directory.
+
+Interceptor implementations must implement _org.jboss.ejb.client.EJBClientInterceptor_
+interface.
+
+Sample configuration:
+
+[source,xml,options="nowrap"]
+----
+<client-interceptors>
+	<interceptor module="org.foo:FooInterceptor:1.0" class="org.foo.FooInterceptor"/>
```

Comment:
This seems to be client-interceptor elsewhere

Reply:
fixed

Reply:
It turned out to be a bug in both client and server interceptors - I have fixed it so that the model and parser are consistent and added the tests. I have also added transformer tests for both client and server interceptors.